### PR TITLE
simplabs monthly update - July 2019

### PR DESCRIPTION
**Pull request by mrmrcoleman**

### CODE REVIEW

File: _posts/2019-07-23-july-monthly-update.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+First announced in the [May monthly update](https://simplabs.com/blog/2019/05/10/may-monthly-update), the ember-intl-analyzer CLI tool which analyzes your app code and finds unused translations for you has received an update.
+
+ember-intl-analyzer now has support or JS decorators and will warn about missing translations.
+
+Check it out here: [https://github.com/simplabs/ember-intl-analyzer](https://github.com/simplabs/ember-intl-analyzer)
+
+#### Other news
+
+In other news [Chris Manson](https://twitter.com/real_ate/) and [Jen Weber](https://twitter.com/jwwweber/) released [another episode](https://twitter.com/real_ate/status/1126760948260204549) of 'May I Ask A Question?'.
```
Comment (mansona) [inline _posts/2019-07-23-july-monthly-update.md]: I would probably just remove this update because it highlights the delay in the update post 🙈 "**May** I Ask A Question" is a pun on the fact that we answer Stack Overflow questions in **May**

File: _posts/2019-07-23-july-monthly-update.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+[EmberFest](https://emberfest.eu/) is coming together nicely.
+
+Taking place in Copenhagen in October, EmberFestEU brings together the European Ember community for two days of talks and activities.
+
+Grab a ticket [here](https://emberfest.eu/) while you still can and keep an eye on the [twitter](https://twitter.com/EmberFest) for updates as the conference takes shape.
+
+#### Chris Manson at Leeds JS
+
+On July 31st [Chris Manson](https://twitter.com/real_ate/) will be speaking about [Empress](https://github.com/hodgesmr/Empress), the bootstrapped blogging template that leverages EmberJS and Github, at Leeds JS.
```
Comment (mansona) [inline _posts/2019-07-23-july-monthly-update.md]: So that link isn't the Empress that I'm working on 😂 (but thanks for pointing out the name clash, I'll reach out to them now) 

The Official Empress is over here: https://github.com/empress and I would use the following as a short intro: "speaking about Empress, which is the technology behind [empress-blog](https://github.com/empress/empress-blog) and [Guidemaker](https://github.com/empress/guidemaker) which powers the current Ember guides" 

Feel free to rephrase as you see fit

**Review by mansona — CHANGES_REQUESTED** [review]
Feel free to keep the thing about "May I Ask a Question" but we need to update the thing about Empress 👍

File: _posts/2019-07-23-july-monthly-update.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+---
+title: "simplabs monthly update - July 2019"
+author: "Mark Coleman"
+github: mrmrcoleman
+twitter: mrmrcoleman
+topic: simplabs
+bio: "Communications and Community Outreach Specialist"
+---
+
+Welcome to the fourth installment of our monthly update. This one is  later than expected but still packed with all the events and activities that have been happening at simplabs along with the things we're looking forward to. Enjoy.
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
Welcome to the fourth installment of our monthly update. This one is later than expected but still packed with all the events and activities that have been happening at simplabs along with the things we're looking forward to. Enjoy.
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+<!--break-->
+
+## News
+
+#### New simplabs website
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### New simplabs website
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+We recently recreated our website in [Glimmer](https://glimmerjs.com/) which describes itself as "Fast and light-weight UI components for the web, with the attention to detail you've come to expect from Ember."
+
+Feel free to take a look around!
+
+#### New blog post from Tobias Bieniek
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### New blog post from Tobias Bieniek
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+[Tobias Bieniek](https://twitter.com/TobiasBieniek) recently released a new blog post entitled "Sentry error reporting for Ember.js" which was positively received.
+
+Check it out [here](https://simplabs.com/blog/2019/07/15/sentry-and-ember/)
+
+#### New version of ember-intl-analyzer released
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### New version of ember-intl-analyzer released
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+First announced in the [May monthly update](https://simplabs.com/blog/2019/05/10/may-monthly-update), the ember-intl-analyzer CLI tool which analyzes your app code and finds unused translations for you has received an update.
+
+ember-intl-analyzer now has support or JS decorators and will warn about missing translations.
+
+Check it out here: [https://github.com/simplabs/ember-intl-analyzer](https://github.com/simplabs/ember-intl-analyzer)
+
+#### Other news
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### Other news
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+In other news [Chris Manson](https://twitter.com/real_ate/) and [Jen Weber](https://twitter.com/jwwweber/) released [another episode](https://twitter.com/real_ate/status/1126760948260204549) of 'May I Ask A Question?'.
+
+[Ricardo Mendes](https://twitter.com/locks) released a new episode of the Conversas em Código podcast, this time talking about [Internationalization, Flags, and Inputs](https://trello.com/c/8W25cdsV/28-episode-19-of-locks-portuguese-podcast).
+
+## Upcoming Events
+
+_We're always interested in new speakers for the various meetups we're involved in. If you'd like to give a talk, [please get in touch.](/contact/)_
+
+#### EmberFest - October 17th & 18th
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### EmberFest - October 17th & 18th
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+[EmberFest](https://emberfest.eu/) is coming together nicely.
+
+Taking place in Copenhagen in October, EmberFestEU brings together the European Ember community for two days of talks and activities.
+
+Grab a ticket [here](https://emberfest.eu/) while you still can and keep an eye on the [twitter](https://twitter.com/EmberFest) for updates as the conference takes shape.
+
+#### Chris Manson at Leeds JS
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### Chris Manson at Leeds JS
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+On July 31st [Chris Manson](https://twitter.com/real_ate/) will be speaking about [Empress](https://github.com/hodgesmr/Empress), the bootstrapped blogging template that leverages EmberJS and Github, at Leeds JS.
+
+Check the full event details [here](https://leedsjs.com/events/2019-07-31/).
+
+## Past Events
+
+#### Ember.js Berlin - Javascript for Beginners
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### Ember.js Berlin - Javascript for Beginners
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+
+On May 24th and 25th [Jessica Jordan](https://twitter.com/jjordan_dev) ran a very succcessful Javascript for beginners workshop. in Berlin.
+
+Day 1 was an installation party to make sure that everybody was set up for day 2 when attendees were guided through creating their first web application using EmberJS! The workshop is open to women and those who are non-binary, and you can sign up using the links below!
+
+#### Jessica Jordan at JSConfEU
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### Jessica Jordan at JSConfEU
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+
+![JsConfEU](/assets/images/posts/2019-05-10-may-monthly-update/jsconfeu.png)
+
+On the 1st of June [Jessica Jordan](https://twitter.com/jjordan_dev) spoke at [JSConfEU](https://2019.jsconf.eu/) with her talk "Crafting Comics for Literally Everyone" in which she highlighted how you can craft an immersive webcomic experience that is accessible for blind, visually-impaired and sighted readers alike!
+
+#### Ricardo Mendes talking at Commit Porto
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### Ricardo Mendes talking at Commit Porto
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+
+On the 22nd of June [Ricardo Mendes](https://twitter.com/locks) gave his talk "Thriving through the hype cycle: an Ember.js story" at [CommitPorto](https://commitporto.com/).
+
+#### Jessica speaking at FullStackCon London
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
### Jessica speaking at FullStackCon London
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+
+Welcome to the fourth installment of our monthly update. This one is  later than expected but still packed with all the events and activities that have been happening at simplabs along with the things we're looking forward to. Enjoy.
+
+<!--break-->
+
+## News
+
+#### New simplabs website
+
+We recently recreated our website in [Glimmer](https://glimmerjs.com/) which describes itself as "Fast and light-weight UI components for the web, with the attention to detail you've come to expect from Ember."
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
We recently recreated our website in [Glimmer.js](https://glimmerjs.com/), the fast and light-weight UI components library, based on Ember.js' rendering engine. We will have a blog post online going into more details soon.
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+
+[Tobias Bieniek](https://twitter.com/TobiasBieniek) recently released a new blog post entitled "Sentry error reporting for Ember.js" which was positively received.
+
+Check it out [here](https://simplabs.com/blog/2019/07/15/sentry-and-ember/)
+
+#### New version of ember-intl-analyzer released
+
+First announced in the [May monthly update](https://simplabs.com/blog/2019/05/10/may-monthly-update), the ember-intl-analyzer CLI tool which analyzes your app code and finds unused translations for you has received an update.
+
+ember-intl-analyzer now has support or JS decorators and will warn about missing translations.
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
ember-intl-analyzer now has support for JS decorators and will warn about missing translations.
```

File: _posts/2019-07-23-july-monthly-update.md (hunk 8 above)
Comment (marcoow) [reply _posts/2019-07-23-july-monthly-update.md]: We should also be saying that the CfP closes on August 1st so people should get their submissions in now. Also we announced that https://twitter.com/chancancode will be speaking.

File: _posts/2019-07-23-july-monthly-update.md [hunk 16]
```diff
@@ -0,0 +1,80 @@
...
+
+On July 31st [Chris Manson](https://twitter.com/real_ate/) will be speaking about [Empress](https://github.com/hodgesmr/Empress), the bootstrapped blogging template that leverages EmberJS and Github, at Leeds JS.
+
+Check the full event details [here](https://leedsjs.com/events/2019-07-31/).
+
+## Past Events
+
+#### Ember.js Berlin - Javascript for Beginners
+
+On May 24th and 25th [Jessica Jordan](https://twitter.com/jjordan_dev) ran a very succcessful Javascript for beginners workshop. in Berlin.
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]: actually @jessica-jordan @sdebarros and @mansona

Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
On May 24th and 25th [Jessica Jordan](https://twitter.com/jjordan_dev) ran a very succcessful Javascript for beginners workshop in Berlin.
```

File: _posts/2019-07-23-july-monthly-update.md [hunk 17]
```diff
@@ -0,0 +1,80 @@
...
+
+Day 1 was an installation party to make sure that everybody was set up for day 2 when attendees were guided through creating their first web application using EmberJS! The workshop is open to women and those who are non-binary, and you can sign up using the links below!
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]: Hm, this doesn't really make sense. Also, we should be saying that we'll do it again in the future.

File: _posts/2019-07-23-july-monthly-update.md [hunk 18]
```diff
@@ -0,0 +1,80 @@
...
+
+#### Jessica Jordan at JSConfEU
+
+![JsConfEU](/assets/images/posts/2019-05-10-may-monthly-update/jsconfeu.png)
+
+On the 1st of June [Jessica Jordan](https://twitter.com/jjordan_dev) spoke at [JSConfEU](https://2019.jsconf.eu/) with her talk "Crafting Comics for Literally Everyone" in which she highlighted how you can craft an immersive webcomic experience that is accessible for blind, visually-impaired and sighted readers alike!
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]: We should link here (1st video at the top): https://simplabs.com/talks/

**Review by marcoow — CHANGES_REQUESTED** [review]

File: _posts/2019-07-23-july-monthly-update.md (hunk 1 above)
Comment (mrmrcoleman) [reply _posts/2019-07-23-july-monthly-update.md]: Aha, ok understood. Will remove.

File: _posts/2019-07-23-july-monthly-update.md (hunk 17 above)
Comment (mrmrcoleman) [reply _posts/2019-07-23-july-monthly-update.md]: Ah my bad, I'll fix that now.

File: _posts/2019-07-23-july-monthly-update.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+
+You can watch the whole talk [here.](https://www.youtube.com/watch?v=BPmuR4mAQaw)
+
+### Ricardo Mendes talking at Commit Porto
+
+On the 22nd of June [Ricardo Mendes](https://twitter.com/locks) gave his talk "Thriving through the hype cycle: an Ember.js story" at [CommitPorto](https://commitporto.com/).
+
+### Jessica speaking at FullStackCon London
+
+And last but not least, [Jessica Jordan](https://twitter.com/jjordan_dev) spoke at [FullStackConf](https://skillsmatter.com/conferences/11213-fullstack-london-2019-the-conference-on-javascript-node-and-internet-of-things) in London with her talk "Crafting Comics or Literally Everyone".
```
Comment (mansona) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
And last but not least, [Jessica Jordan](https://twitter.com/jjordan_dev) spoke at [FullStackConf](https://skillsmatter.com/conferences/11213-fullstack-london-2019-the-conference-on-javascript-node-and-internet-of-things) in London with her talk "Crafting Comics for Literally Everyone".
```
I never notice typos but I saw this when I was double checking my parts

**Review by mansona — CHANGES_REQUESTED** [review]

**Review by mansona — APPROVED** [review]

File: _posts/2019-07-23-july-monthly-update.md [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+
+_We're always interested in new speakers for the various meetups we're involved in. If you'd like to give a talk, [please get in touch.](/contact/)_
+
+### EmberFest - October 17th & 18th
+
+[EmberFest](https://emberfest.eu/) is coming together nicely.
+
+Taking place in Copenhagen in October, EmberFestEU brings together the European Ember community for two days of talks and activities.
+
+The CFP closes on August 1st so if you'd like to give a presentation remember to get your submission on time [here](https://cfp.emberfest.eu/events/emberfest-2019).
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
The CFP closes on August 1st so if you'd like to give a presentation remember to get your submission in on time [here](https://cfp.emberfest.eu/events/emberfest-2019).
```

?

File: _posts/2019-07-23-july-monthly-update.md [hunk 21]
```diff
@@ -0,0 +1,94 @@
...
+
+### Upcoming webinar - The ultimate marketers guide to modern web apps and why users are demanding more
+
+On August 12th at 6pm UTC+2 we're running a webinar to help forward-looking marketers understand recent changes in web technology and how they can be used to create incredible experiences for their users.
+
+From the site: _"The way that web apps are built has changed dramatically in recent years. Changes in devices, browsers and usage patterns have created a wave of innovation which is waiting to be used by forward looking marketing departments._
+
+_After this webinar you will understand why these changes are occurring, how technology is responding to this shift and crucially, how you can use these technologies to build incredible experiences for your users."_
+
+You can sign up [here.](https://simplabs.com/webinars/modern-web/)
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]:
```suggestion
You can sign up [here.](/webinars/modern-web/)
```

**Review by marcoow — APPROVED** [review]

File: _posts/2019-07-23-july-monthly-update.md [hunk 22]
```diff
@@ -0,0 +1,100 @@
...
+
+Welcome to the fourth installment of our monthly update. This one is later than expected but still packed with all the events and activities that have been happening at simplabs along with the things we're looking forward to. Enjoy.
+
+<!--break-->
+
+## News
+
+### Upcoming webinar - The ultimate marketers guide to modern web apps and why users are demanding more
+
+On August 12th at 6pm UTC+2 we're running a webinar to help forward-looking marketers understand recent changes in web technology and how they can be used to create incredible experiences for their users.
```
Comment (Turbo87) [inline _posts/2019-07-23-july-monthly-update.md]: `UTC+2` is hard to parse for me. can we replace that with `CEST`?

```suggestion
On August 12th at 6pm CEST we're running a webinar to help forward-looking marketers understand recent changes in web technology and how they can be used to create incredible experiences for their users.
```

**Comment by marcoow** [comment]
@pichfl can you merge this once you're happy?

**Comment by pichfl** [comment]
@marcoow I added the webinar ad to the post so it has little more color, but found that I need to fix a layout bug to do so. See #641 

Will rebase and publish once that is merged.

File: _posts/2019-07-23-july-monthly-update.md [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+bio: 'Communications and Community Outreach Specialist'
+---
+
+Welcome to the fourth installment of our monthly update. This one is later than expected but still packed with all the events and activities that have been happening at simplabs along with the things we're looking forward to. Enjoy.
+
+<!--break-->
+
+## News
+
+[![Upcoming webinar - The ultimate marketers guide to modern web apps and why users are demanding more](/assets/images/posts/2019-07-23-july-monthly-update/simplabs-webinar-modern-web-apps-for-marketers.png)](/webinars/modern-web/)
```
Comment (marcoow) [inline _posts/2019-07-23-july-monthly-update.md]: clicking on this takes me to the home page 🤔

Comment (pichfl) [reply _posts/2019-07-23-july-monthly-update.md]: Can reproduce. I think I uncovered an error in the way we handle internal links:

https://github.com/simplabs/simplabs.github.io/blob/c3d3099785a95c4e0e7de2a3b74331a3cc91cad0/src/ui/components/Simplabs/component.ts#L143-L147

that `target` in L147 should be `link`

See PR #643

**Comment by pichfl** [comment]
As discussed with @mrmrcoleman we will publish this first thing tomorrow. Updating the date of the blog post.
